Sources/MuxUploadSDK/Internal Utilities/UploadEvent.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  UploadEvent.swift
+//  
+//
+//  Created by Liam Lindner on 3/22/23.
+//
+
+import Foundation
+import UIKit
+
+struct UploadEvent: Codable {
+    var type = "upload"
+    var start_time: TimeInterval
+    var end_time: TimeInterval
+    var file_size: UInt64
+    var video_duration: Double
+
+    var sdk_version: String
+
+    var os_name: String
+    var os_version: String
+
+    var device_model: String
+
+    var app_name: String?
+    var app_version: String?
+
+    var regionCode: String?
+
+    init(start_time: TimeInterval, end_time: TimeInterval, file_size: UInt64, video_duration: Double) {
+        let locale = Locale.current
+        let device = UIDevice.current
+
+        self.start_time = start_time
+        self.end_time = end_time
+        self.file_size = file_size
+        self.video_duration = video_duration
+
+        self.sdk_version = "0.2.0" // TODO Read from properties
```

Comment:
Any recommendations for doing this programmatically?

Reply:
Can reference what's [here](https://github.com/muxinc/swift-upload-sdk/blob/74671959fa979b70785fdae6934da09cc1151a53/Sources/MuxUploadSDK/Public%20API/Version.swift#L19) to remove the hardcoded version as an interim solution.